jwe.go

```diff
@@ -216,7 +216,8 @@ func parseEncryptedCompact(input string) (*JsonWebEncryption, error) {
 
 // CompactSerialize serializes an object using the compact serialization format.
 func (obj JsonWebEncryption) CompactSerialize() (string, error) {
-	if len(obj.recipients) > 1 || obj.unprotected != nil || obj.recipients[0].header != nil {
+	if len(obj.recipients) > 1 || obj.unprotected != nil ||
```

Comment:
Do we need to check that obj.recipients has >0 entries to avoid an `index out of range` panic, too?